templates.py

```diff
@@ -3,13 +3,13 @@
 <html lang="en">
```

Comment:
Since this template is used only by the admin interface, perhaps `templates.py` should be moved to `admin`.